client/html/blocks/header.html

```diff
@@ -1,5 +1,5 @@
 <header class="header page__header color_@@color">
-    <a href="/" class="header__menu-button button">
+    <a href="" ng-click="isNavHide = !isNavHide" class="header__menu-button button">
```

Comment:
Move isNavHide assignment to controller. This kind of logic should be done there controller.